src/test/java/nl/atos/ontwikkelstraat/FormControllerSteps.java

```diff
@@ -0,0 +1,146 @@
+        //System.out.println(webdriver.findElement(By.id("name")).getAttribute("value"));
+    }
+
+
+    @And("^I fill in my lastName$")
+    public void i_fill_in_my_lastName() throws Throwable {
+        formData.setSurname("Johnson");
+        webdriver.findElement(By.id("surname")).sendKeys("Johnson");
+        assertEquals("Johnson", webdriver.findElement(By.id("surname")).getAttribute("value"));
+    }
+
+    @And("^I fill in my StreetName$")
+    public void i_fill_in_my_StreetName() throws Throwable {
+        formData.setStreetName("hoofdstraat");
+        webdriver.findElement(By.id("streetName")).sendKeys("hoofdstraat");
+        assertEquals("hoofdstraat", webdriver.findElement(By.id("streetName")).getAttribute("value"));
+    }
+
+    @And("^I fill in my HouseNumber$")
+    public void i_fill_in_my_HouseNumber() throws Throwable {
+        formData.setHouseNumber(134);
+        webdriver.findElement(By.id("houseNumber")).sendKeys("134");
+        assertEquals("134", webdriver.findElement(By.id("houseNumber")).getAttribute("value"));
+    }
+
+    @And("^I fill in my Zipcode$")
+    public void i_fill_in_my_Zipcode() throws Throwable {
+        formData.setZipCode("9485SL");
+        webdriver.findElement(By.id("zipCode")).sendKeys("9485SL");
+        assertEquals("9485SL", webdriver.findElement(By.id("zipCode")).getAttribute("value"));
+    }
+
+    @And("^I hit the submit button$")
+    public void i_hit_the_submit_button() throws Throwable {
+        JavascriptExecutor jse = (JavascriptExecutor) webdriver;
+        jse.executeScript("window.scrollBy(0,250)", "");
+        webdriver.findElement(By.id("testSubmit")).click();
+    }
+
+    @Then("^I should see my name and address data on the next page and it is also saved$")
+    public void i_should_see_my_name_and_address_data_on_the_next_page_and_it_is_also_saved() throws Throwable {
+        assertEquals(formData.getName(), webdriver.findElement(By.id("name")).getAttribute("value"));
+        assertEquals(formData.getSurname(), webdriver.findElement(By.id("surname")).getAttribute("value"));
+        assertEquals(formData.getStreetName(), webdriver.findElement(By.id("streetName")).getAttribute("value"));
+        assertEquals(formData.getHouseNumber().toString(), webdriver.findElement(By.id("houseNumber")).getAttribute("value"));
+        assertEquals(formData.getZipCode(), webdriver.findElement(By.id("zipCode")).getAttribute("value"));
+    }
+
+
+    @And("^The browser should be closed$")
+    public void theBrowserShouldBeClosed() throws Throwable {
+        // Write code here that turns the phrase above into concrete actions
+        //webdriver.close();
+//        webdriver.manage().timeouts().setScriptTimeout(10, TimeUnit.SECONDS);
+//        webdriver.close();
+    }
+
+    @And("^I fill in my lastName with \"([^\"]*)\"$")
+    public void i_fill_in_my_lastName_with(String lastName) throws Throwable {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/vinsterdamus/Ontwikkelstraat/file/2345692422/issues/source?bid=3138095&fileBranchId=3208141#l110)